### DISCUSSION TITLE
update the `updated` field only on certain column updates

### DISCUSSION THREAD

**Discussion by Amir-Zouerami** [pocketbase/pocketbase#3911] [Q&A] [answered]
for a blog post I have a `viewCounter` column which updates on every page load to increment it. Is there a way for me to stop the `updated` column from updating in this occasion? I only want to update the `updated` column on `blogContent` column updates (when the blog post content changes, not the metadata like view counter).

**Comment by ganigeorgiev** [comment, marked as answer]
No, at the moment it is not possible to change the default system fields behavior, at least not easily.

How is the `viewCounter` updated - via the web APIs/SDK or via hook(s)?

If it is via hook, one workaround could be to perform a [raw db update](https://pocketbase.io/docs/js-database/#executing-queries) but note that this will not trigger the realtime events. Here is one such example with the JS hooks:

```js
$app.dao().db()
  .newQuery("UPDATE example SET viewCounter = (viewCounter + 1) WHERE id={:id}")
  .bind({ id: "YOUR_ID" })
  .execute()
```

**Reply by Amir-Zouerami** [reply]
it's via SDK. thank you for the answer though.
